app/javascript/hyacinth_ui_v1/components/ui/forms/inputs/term_select/TermMenu.js

```diff
@@ -14,65 +9,27 @@ const TermMenu = React.forwardRef((props, ref) => {
     onChange,
     style,
     'aria-labelledby': labeledBy,
-    vocabulary: vocabularyStringKey,
+    vocabulary,
     close,
   } = props;
 
-  const [loading, setLoading] = useState(true);
-  const [displayNewTerm, setDisplayNewTerm] = useState(false);
-  const [vocabulary, setVocabulary] = useState({});
-
-  useEffect(() => {
-    vocabularies.get(vocabularyStringKey).then((res) => {
-      setVocabulary(res.data.vocabulary);
-      setLoading(false);
-    });
-  }, []);
-
   return (
     <div ref={ref} style={{ ...style, width: '100%' }} className={className} aria-labelledby={labeledBy}>
-      { loading && <div className="m-3"><Spinner animation="border" variant="warning" /></div>}
-      {
-        !loading && (
-          <>
-            <Dropdown.Header>
-              {`${vocabulary.label} Controlled Vocabulary`}
-              {
-                !displayNewTerm && (
-                  <span className="float-right">
-                    <AddButton onClick={() => setDisplayNewTerm(true)}> New Term</AddButton>
-                  </span>
-                )
-              }
-            </Dropdown.Header>
-            <Dropdown.Divider />
-
-            {
-              displayNewTerm ? (
-                <div className="px-3">
-                  <TermForm
-                    small
-                    formType="new"
-                    vocabulary={vocabulary}
-                    submitAction={(term) => { onChange(term); setDisplayNewTerm(false); close(); }}
-                    cancelAction={() => setDisplayNewTerm(false)}
-                  />
-                </div>
-              ) : <TermOptions vocabulary={vocabulary.stringKey} onChange={onChange} />
-            }
-          </>
-        )
-      }
+      <TermOptions
+        vocabularyStringKey={vocabulary}
+        onChange={onChange}
+        close={close}
+      />
     </div>
   );
 });
 
 TermMenu.propTypes = {
   vocabulary: PropTypes.string.isRequired,
-  value: PropTypes.shape({
-    prefLabel: PropTypes.string,
-    uri: PropTypes.string,
-  }).isRequired,
+  // value: PropTypes.shape({
```

Comment:
Can we just delete the commented-out block?

Reply:
Yes. Just pushed a change.